modules/loki/main.tf

```diff
@@ -0,0 +1,29 @@
+module "loki" {
+
+  count = var.loki_storage.type == "digitalocean" ? 1 : 0
```

Comment:
why we need to have this check?
the module seems is general and can be used for any storage type